vendor/k8s.io/kubernetes/test/e2e/framework/pods.go

```diff
@@ -292,3 +295,54 @@ func (c *PodClient) PodIsReady(name string) bool {
 	ExpectNoError(err)
 	return podutil.IsPodReady(pod)
 }
+
+// ExecWithOptions executes a command in the specified container,
+// returning stdout, stderr and error. `options` allowed for
+// additional parameters to be passed.
+func (c *PodClient) ExecWithOptions(options ExecOptions, cli clientset.Interface) (string, string, error) {
```

Comment:
Nit: "cli" is a very unusual parameter name for "clientset.Interface", it occurs only once in the entire Kubernetes code base. "cs" is much more common. It's also more common to have it first in the parameter list.